config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  resources :users, only: [:index, :show]
+
+  resources :users do
+    resources :posts, only: [:index, :show]
+  end
```

Comment:
- [ OPTIONAL ] To avoid repeating the user routes, I would highly suggest you refactor this code like this :point_down: 
```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
``` 

```suggestion
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```